apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -38,6 +38,21 @@ model User {
   guild_users          GuildUser[]
   twitter_user         TwitterUser?
   discord_users        DiscordUser[]
+  tips                 Tip[]
+  split_contracts      UserSplitContract[]
```

Comment:
Good idea! Gives us leeway to support multiple per user!

Reply:
yep. idea is to support multiple chains.